### ISSUE TITLE
RawArray doesn't support BitArrays

### ISSUE THREAD

**Issue by RobBlackwell** [davidssmith/RawArray.jl#4]
It would be nice to be able to save BitArrays too, but this fails for me:

```
x = BitArray(10,10)
rawrite(x,"test.ra")
```

**Comment by davidssmith** [comment]
Fixed in #5. 

Once the new version of `LittleEndianBase128` is merged into the official METADATA, then I can bump this package's version and you can get everything via `Pkg.update()`.

If you want the changes sooner, you can run
```
Pkg.checkout("LittleEndianBase128")
Pkg.checkout("RawArray")
```
